### PR TITLE
Fix AppShell handle-errors related to `Avatar` entities

### PR DESCRIPTION
**Pull request by sjmiles**
clip horizontally, remove vestigial static identities handle

### CODE REVIEW

File: shell/app-shell/app-shell.js [hunk 1]
```diff
@@ -200,7 +201,6 @@ const template = ArcsUtils.html`
     <arc-handle arc='{{arc}}' data='{{arcsHandleData}}' options='{{arcsHandleOptions}}' on-change='_onArcsHandleChange'></arc-handle>
     <arc-handle arc='{{arc}}' data='{{identityHandleData}}' options='{{identityHandleOptions}}' on-change='_onIdentityHandleChange'></arc-handle>
     <arc-handle arc='{{arc}}' data='{{identitiesHandleData}}' options='{{identitiesHandleOptions}}' on-change='_onIdentitiesHandleChange'></arc-handle>
-    <arc-handle arc='{{arc}}' data='{{friendsAvatarData}}' options='{{friendsAvatarHandleOptions}}'></arc-handle>
```
Comment (shaper) [inline shell/app-shell/app-shell.js]: Can we also delete lines 308 - 316 in `app-shell.js` or are you planning to put this back in some form later? Note `vr-app-shell.js` also has this `arc-handle`.

Comment (sjmiles) [reply shell/app-shell/app-shell.js]: There are no commented out lines in `app-shell.js` that are there by accident. But I'm getting the message that people are not used to this, so I will change my behavior and be more aggressive about deleting such code.

Comment (shaper) [reply shell/app-shell/app-shell.js]: Those lines aren't commented out -- they build the data that it looked like this handle was the only place using.

**Comment by sjmiles** [comment]
(Btw, seems like there's no way to respond to comments inline, which always confuses me.)

Re: #800, I hope it fixes it, I'm not 100% sure.

Re: why the deletion: this static handle was replace with a dynamic handle created by remote-friends-profile-handles.

File: shell/app-shell/app-shell.js (hunk 1 above)
Comment (sjmiles) [reply shell/app-shell/app-shell.js]: Sorry, my bad, was looking at the wrong thing. You are totally right, will remove.

**Comment by sjmiles** [comment]
Thanks for jumping on this so quickly, Walt. You're the man.

**Pull request merged**